### PR TITLE
Rename get product category constants

### PR DESCRIPTION
**Pull request by danielfrg**

### CODE REVIEW

File: examples/recommendations/detect_and_apply_recommendations.rb [hunk 1]
```diff
@@ -70,10 +70,46 @@ def detect_and_apply_recommendations(customer_id)
...
+  #    client.path.recommendation(customer_id, recommendation_id)
+
+  apply_recommendation_operation = client.operation.apply_recommendation
+  apply_recommendation_operation.resource_name = recommendation_resource
+
+  # Each recommendation type has optional parameters to override the recommended
+  # values. This is an example to override a recommended ad when a
+  # TextAdRecommendation is applied.
+  # For details, please read
+  # https://developers.google.com/google-ads/api/reference/rpc/google.ads.google_ads.v1.services#google.ads.google_ads.v1.services.ApplyRecommendationOperation
```
Comment (mcloonan) [inline examples/recommendations/detect_and_apply_recommendations.rb:87]: This link is a 404

File: examples/recommendations/detect_and_apply_recommendations.rb [hunk 2]
```diff
@@ -70,10 +70,46 @@ def detect_and_apply_recommendations(customer_id)
   end
 end
 
+# [START apply_recommendation]
+def apply_recommendation(client, customer_id, recommendation_resource)
+  # If you have a recommendation_id instead of the resournce_name
```
Comment (mcloonan) [inline examples/recommendations/detect_and_apply_recommendations.rb:75]: typo in resource_name

File: examples/recommendations/detect_and_apply_recommendations.rb [hunk 3]
```diff
@@ -33,34 +33,34 @@ def detect_and_apply_recommendations(customer_id)
   client = Google::Ads::GoogleAds::GoogleAdsClient.new
 
   query = <<~QUERY
-    SELECT recommendation.resource_name
+    SELECT recommendation.resource_name, recommendation.campaign,
+        recommendation.keyword_recommendation
```
Comment (mcloonan) [inline examples/recommendations/detect_and_apply_recommendations.rb:37]: Can we put one field per line?

File: examples/recommendations/detect_and_apply_recommendations.rb [hunk 4]
```diff
@@ -33,34 +33,34 @@ def detect_and_apply_recommendations(customer_id)
...
+        "campaign '#{recommendation.campaign}'."
+      if recommendation.keyword_recommendation
+        keyword = recommendation.keyword_recommendation.keyword
+        puts "\tKeyword = '#{keyword.text}'\n\ttype = '#{keyword.match_type}'"
       end
+
+      apply_recommendation(client, customer_id, recommendation.resource_name)
     end
 
     if i < NUMBER_OF_RUNS - 1
```
Comment (mcloonan) [inline examples/recommendations/detect_and_apply_recommendations.rb:66]: Why does this still exist if we're reducing the number of runs to 1?

**Review by mcloonan — CHANGES_REQUESTED** [review]

File: examples/recommendations/detect_and_apply_recommendations.rb (hunk 4 above)
Comment (danielfrg) [reply examples/recommendations/detect_and_apply_recommendations.rb:66]: Sorry, ignore this changes!

**Pull request closed without merging**